src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -374,17 +382,36 @@ public StudyService(
                     .toUriString();
 
             return webClient.put()
-                    .uri(loadFlowServerBaseUri + path)
-                    .retrieve()
-                    .bodyToMono(Void.class);
+                .uri(loadFlowServerBaseUri + path)
+                .retrieve()
+                .bodyToMono(String.class)
+                .flatMap(e -> studyRepository.updateLoadFlowResult(studyName, userId, jsonToLoadFlowResult(e)))
+                .doOnError(e -> studyRepository.updateLoadFlowState(studyName, userId, LoadFlowResult.LoadFlowStatus.NOT_DONE).block());
+
         }).doOnSuccess(s ->
             studyUpdatePublisher.onNext(MessageBuilder.withPayload("")
                 .setHeader(STUDY_NAME, studyName)
                 .setHeader(UPDATE_TYPE, UPDATE_TYPE_LOADFLOW)
-                .build())
+                .build()
+            )
         );
     }
 
+    private LoadFlowResult jsonToLoadFlowResult(String strLfResult) {
+        try {
+            String strStatus = JsonPathUtils.evaluate(strLfResult, "$.metrics.network_0_status");
```

Comment:
@JacquesBorsenberger here you hard code that result is from main connected component, but it should be up to the modele to decide about the status. There is a "ok" attribute for that in loadflow results.

Reply:
done